testsuite/features/support/pretty_formatter.rb

```diff
@@ -1,8 +1,8 @@
-# Copyright (c) 2013-2019 SUSE LLC.
+# Copyright (c) 2021 SUSE LLC.
```

Comment:
2013-2021, no ?

Reply:
Actually  I created it in 2019. So 2019-2021